modules/flatbuffers/24.3.7/presubmit.yml

```diff
@@ -0,0 +1,31 @@
+---
+buildifier: latest
+bazel: 6.4.0
```

Comment:
Can you replace with `6.x` to fetch the latest minor release for Bazel 6? (currently 6.5.0)

This module version doesn't work with 7.1?